src/Opdex.Platform.Infrastructure.Abstractions/Data/Queries/VaultGovernances/Votes/VaultProposalVotesCursor.cs

```diff
@@ -0,0 +1,93 @@
+using Opdex.Platform.Common.Enums;
+using Opdex.Platform.Common.Extensions;
+using Opdex.Platform.Common.Models;
+using System;
+using System.Text;
+
+namespace Opdex.Platform.Infrastructure.Abstractions.Data.Queries.VaultGovernances.Votes;
+
+public class VaultProposalVotesCursor : Cursor<ulong>
+{
+    public VaultProposalVotesCursor(ulong proposalId, Address voter, bool includeZeroBalances, SortDirectionType sortDirection, uint limit, PagingDirection pagingDirection, ulong pointer)
+        : base(sortDirection, limit, pagingDirection, pointer)
+    {
+        ProposalId = proposalId;
+        Voter = voter;
+        IncludeZeroBalances = includeZeroBalances;
+    }
+
+    public ulong ProposalId { get; }
+    public Address Voter { get; }
+    public bool IncludeZeroBalances { get; }
+
+    /// <inheritdoc />
+    public override string ToString()
+    {
+        var pointerBytes = Encoding.UTF8.GetBytes(Pointer.ToString());
+        var encodedPointer = Convert.ToBase64String(pointerBytes);
+        return $"proposalId:{ProposalId};voter:{Voter};includeZeroBalances:{IncludeZeroBalances};direction:{SortDirection};limit:{Limit};paging:{PagingDirection};pointer:{encodedPointer};";
+    }
+
+    /// <inheritdoc />
+    public override Cursor<ulong> Turn(PagingDirection direction, ulong pointer)
+    {
+        if (!direction.IsValid()) throw new ArgumentOutOfRangeException(nameof(direction), "Invalid paging direction.");
+        if (pointer == Pointer) throw new ArgumentOutOfRangeException(nameof(pointer), "Cannot paginate with an identical id.");
+
+        return new VaultProposalVotesCursor(ProposalId, Voter, IncludeZeroBalances, SortDirection, Limit, direction, pointer);
+    }
+
+    /// <summary>
+    /// Parses a stringified version of the cursor
+    /// </summary>
+    /// <param name="raw">Stringified cursor</param>
+    /// <param name="cursor">Parsed cursor</param>
+    /// <returns>True if the value could be parsed, otherwise false</returns>
+    public static bool TryParse(string raw, out VaultProposalVotesCursor cursor)
+    {
+        cursor = null;
+
+        if (raw is null) return false;
+
+        var values = ToDictionary(raw);
+
+        if (!TryGetCursorProperty<ulong>(values, "proposalId", out var proposalId)) return false;
```

Comment:
Same comment, this is optional so I think we shouldn't return false if not found